Source/Syrup/Player/SyrupPlayerCharacter.h

```diff
@@ -0,0 +1,72 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Character.h"
+#include "Camera/CameraComponent.h"
+#include "SyrupPlayerCharacter.generated.h"
+
+/* \/ ===================== \/ *\
+|  \/ ASyrupPlayerCharacter \/  |
+\* \/ ===================== \/ */
+UCLASS()
+class SYRUP_API ASyrupPlayerCharacter : public ACharacter
+{
+	GENERATED_BODY()
+
+	/* ---------------- *\
+	\* \/ Components \/ */
+
+private:
+	UPROPERTY(VisibleAnywhere, Category = "Components")
+	UCameraComponent* Camera;
+
+	UPROPERTY(VisibleAnywhere, Category = "Components")
+	USceneComponent* CameraRoot;
+	
+	/* /\ Components /\ *\
+	\* ---------------- */
+
+
+public:
+	// Sets default AxisValues for this character's properties
+	ASyrupPlayerCharacter();
+
+	/* --------------- *\
+	\* \/ Variables \/ */
```

Comment:
Variables is not a good section comment. It describes nothing, either change it to something relating to the function of the variables or remove it entirely. A good category you could use would be Input, and then you could group the variables with the input functions.